src/components/About.vue

```diff
@@ -1,22 +1,22 @@
 <template>
-  <div id="aboutSection">
+  <div
+    id="aboutSection"
+    class="Section"
```

Comment:
```suggestion
    class="section"
```
クラス名とかは小文字で始まることが多いです。
https://designsupply-web.com/developmentlab/4052/

Reply:
sectionというタグがあるので、この命名は微妙かもですが